_posts/2024-10-28-matriz-riesgo-seccion-cliente.md

```
@@ -0,0 +1,22 @@
+tags: [ riesgo, creditos , clientes, ejecutivos, operaciones ]
+comments: false
+author: Erick Burgos
+---
+
+## Nuevas Secciones en el KPI de cliente
+
+
+Nos complace anunciar una mejora significativa en la Matriz de Riesgo de Advance: la separación de los KPI del cliente en tres apartados distintos: `Casos Civiles`, `Documentos` y `Cesiones`. Esta reestructuración permitirá a los usuarios acceder de manera más rápida y eficiente a la información relevante, facilitando un análisis más claro y una mejor toma de decisiones. Acompáñanos a explorar cómo estas nuevas categorías optimizan tu experiencia en Advance.
+
+
+<video width="100%"  controls autoplay loop>
```

Markdownlint

Element: video
Inline HTML

(MD033, no-inline-html)